src/libraries/System.Private.Xml/src/System/Xml/Schema/ContentValidator.cs

```diff
@@ -1536,50 +1534,6 @@ private void CheckUniqueParticleAttribution(BitSet curpos)
             // now convert transition table to array
             return (int[][])transitionTable.ToArray(typeof(int[]));
         }
-
-#if DEBUG
-        private void Dump(StringBuilder bb, BitSet[] followpos, int[][] transitionTable)
```

Comment:
I've occasionally found the DEBUG code useful when debugging some cryptic bugs in XML. I'd prefer we left it in place

Reply:
Restored. Thanks for the feedback!